view/statistics-files-pending.js

```diff
@@ -102,8 +103,20 @@ exports['statistics-main'] = function () {
 			this.processingStepsMeta.frontDesk ?
 					generateRow({ class: 'statistics-table-sub-header statistics-table-sub-header-success' },
 						_("Pending for withdraw at Front Desk"), services, function (data) {
-							data = data.atPartB.getBySKeyPath(resolveFullStepPath('frontDesk'));
-							return data ? data._pending : null;
+							var result = null;
+							if (!getFrontDeskNames().size) return null;
+							getFrontDeskNames().forEach(function (name) {
+								var value = data.atPartB.getBySKeyPath(resolveFullStepPath(name));
+								if (value) {
+									if (!result) {
+										result = value._pending;
+									} else {
+										result.value += value.pending;
```

Comment:
I think you modify here object which should not be modified, you take object that resolves value for single step, and try to add there value from other step. It should technically crash, if it doesn't then it's even worse (data gets written to server?)